packages/api/src/functions/graphql.ts

```diff
@@ -95,7 +95,7 @@ export const createGraphQLHandler = ({
   onHealthCheck,
   ...options
 }: GraphQLHandlerOptions = {}) => {
-  const isDevEnv = process.env.NODE_ENV !== 'production'
+  const isDevEnv = process.env.NODE_ENV === 'development'
```

Comment:
I thought about adding `|| process.env.NODE_ENV === 'test'` as well, but the playground isn't needed when running tests, right? And neither is introspection I guess. Maybe we want debug. What do you all think?

Reply:
Agreed!